evalml/pipelines/utils.py

```diff
@@ -140,6 +148,26 @@ class GeneratedPipeline(base_class):
     return GeneratedPipeline
 
 
+def get_generated_pipeline(problem_type):
+    """Returns the class for the generated pipeline based on the problem type
+
+    Arguments:
+        problem_type (Problem_Type): The problem_type that the pipeline is for
+    """
+    if problem_type == ProblemTypes.BINARY:
+        return GeneratedPipelineBinary
+    elif problem_type == ProblemTypes.MULTICLASS:
+        return GeneratedPipelineMulticlass
+    elif problem_type == ProblemTypes.REGRESSION:
+        return GeneratedPipelineRegression
+    elif problem_type == ProblemTypes.TIME_SERIES_REGRESSION:
+        return GeneratedPipelineTimeSeriesRegression
+    elif problem_type == ProblemTypes.TIME_SERIES_BINARY:
+        return GeneratedPipelineTimeSeriesBinary
+    else:
+        return GeneratedPipelineTimeSeriesMulticlass
```

Comment:
Maybe it'd be to define a dictionary mapping of problem type to class and then just return the value, rather than all of these if conditions? :P Also works fine now that else returns `GeneratedPipelineTimeSeriesMulticlass` but we're more likely to run into trouble if/when we add a new pipeline class lol